docs/portal.properties-Reference.md

```diff
@@ -416,3 +417,44 @@ For more information on how Ehcache is implemented in cBioPortal refer to the [C
 ```
 skin.show_gsva=true
 ```
+
+# Request Body Compression
+
+## Background
+Some REST endpoints that the cBioPortal frontend uses have request bodies that scale as your dataset increases. In
+portals where users commonly query more than 100,000 samples, we found that some of these request bodies could
+get as large as 20 Mb. These large request bodies pose a significant problem for users with poor upload speeds - some
+users experienced upload times of more than five minutes for these requests. Request body compression is our solution to
```

Comment:
maybe "short-term solution" (because when sample lists are 300000 samples we will again see this problem even with compression)